contrib/pg_tde/t/wal_encrypt.pl

```diff
@@ -15,8 +15,8 @@
 $node->init;
 $node->append_conf('postgresql.conf', "shared_preload_libraries = 'pg_tde'");
 $node->append_conf('postgresql.conf', "wal_level = 'logical'");
-# NOT testing that it can't start: the test framework doesn't have an easy way to do this
-#$node->append_conf('postgresql.conf', "pg_tde.wal_encrypt = 1"});
+# NOTE: Testing that it can't start: the test framework doesn't have an easy way to do this
```

Comment:
`NOTE: Not testing` maybe?

Reply:
Seems like I fucked it up worse when trying to fix it.

Reply:
Wrote something better.